.github/shared/src/array.js

```diff
@@ -31,3 +31,23 @@ export async function flatMapAsync(array, asyncMapper) {
 export async function mapAsync(array, asyncMapper) {
   return Promise.all(array.map(asyncMapper));
 }
+
+/**
+ * @template T
+ * @param {T[]} array
+ * @param {T[]} values
+ * @returns
+ */
+export function includesAll(array, values) {
```

Comment:
```suggestion
export function includesEvery(array, values) {
```

I may have suggested `includesAll`, but I think `includesEvery` may be better for symmetry with `array.every`.